src/MSBuild/CommandLineSwitches.cs

```diff
@@ -94,9 +94,7 @@ internal enum ParameterizedSwitch
             FileLoggerParameters7,
             FileLoggerParameters8,
             FileLoggerParameters9,
-#if FEATURE_NODE_REUSE
-            NodeReuse,
-#endif
+            NodeReuse, // Should not hide this option when FEATURE_NODE_REUSE is turned off, otherwise existing build scripts which pass /NodeReuse:False would start failing even though they're fully compatible with a build without FEATURE_NODE_REUSE
```

Comment:
I think you can drop this comment; it'll be in the commit description and in the handling code.